modules/bar/window_title/index.ts

```diff
@@ -177,35 +178,29 @@ const ClientTitle = (): BarBoxChild => {
                     truncation_size.bind('value'),
                 ],
                 (client, useCustomTitle, useClassName, showLabel, showIcon, truncate, truncationSize) => {
+                    const children: Label<never>[] = [];
```

Comment:
```suggestion
                    const children: Label<Child>[] = [];
```
Let's assign the right type to the Label.